packages/workbox-precaching/src/index.js

```diff
@@ -16,8 +16,10 @@
 /**
  * # workbox-precaching
  *
- * The precaching module provides helpers that make it easy to cache files
- * during the install step of your service worker.
+ * A lover-level service worker library to precache a manifest of URLs.
+ *
+ * It's recommended that you use the higher-level [`workbox-sw`](../workbox-sw)
```

Comment:
I copied this from Jeff's README edit. 

@jeffposnick @gauntface Please come to agreement.